BlueprintUI/Sources/Environment/Adapted.swift

```diff
@@ -0,0 +1,41 @@
+import CoreGraphics
+
+/// Wraps an element tree with a modified environment.
+///
+/// By specifying environmental values with this element, all child elements nested
+/// will automatically inherit those values automatically. Values can be changed
+/// anywhere in a sub-tree by inserting another `Adapted` element.
+public struct Adapted: Element {
```

Comment:
`AdaptedEnvironmentElement`? Not super clear from the name what this does or why

Reply:
This is the element you use to actually put values in the environment. So the idea was that it _adapts_ the environment for all of its children.

Reply:
Renamed to `AdaptedEnvironment`!